modules/node_modules/@ncigdc/components/MostAffectedCases.js

```diff
@@ -125,10 +127,10 @@ const MostAffectedCases = ({
                   dataCategorySummary[DATA_CATEGORIES[k].full] ? (
                     <a
                       href={
-                        `/search/f?filters=${makeFilter([
+                        `/repository?filters=${JSURL.stringify(makeFilter([
```

Comment:
if you use the `RepositoryFileLink` component instead, wouldn't need to manually call JSURL.stringify example https://github.com/NCI-GDC/portal-ui/blob/relay/modules/node_modules/%40ncigdc/containers/CaseTr.js#L53